docs/docs/edsl.md

```diff
@@ -455,24 +455,22 @@ Tensor csum(const Tensor& I) {
 Alternatively, we could write `k = i - j` for `j` non-negative as an alternative
 way of forcing `k` to be no larger than `i`. Then in summation notation we have:
 
-```math
+$$
 \Large
-\textcolor{red}{O[i]}
-\textcolor{yellow}{=}
-\textcolor{green}{\sum}\textcolor{magenta}{_{0 \leq j}}
-\textcolor{cyan}{I[i - j]}
-```
+\color{red} O[i]
+\color{yellow} =
+\color{green} \sum_{\color{magenta} 0 \leq j} 
+\color{cyan} I[i - j]
+$$
+
 
-```math
+$$
 \Large
-\texttt{
-  if (\textcolor{magenta}{j < N}) \{{
-    \textcolor{red}{O(n)}
-    \textcolor{green}{+=}
-    \textcolor{cyan}{I(i - j)};
-  \}}
+
+  \text{if} ( \color{magenta}j < N \color{black} ) \color{red} O(n) \color{green}{+=} \color{cyan} I(i - j) \color{black};
```

Comment:
If at all possible, I'd like to avoid smashing multiple lines down into a single one, it makes it much harder to figure out what's going on (to me anyways).